CPS-0009/README.md

```diff
@@ -0,0 +1,98 @@
+  - https://github.com/cardano-foundation/CIPs/pull/611
+Created: 2023-10-20
+---
+
+## Abstract
+
+The introduction of native tokens in Cardano has introduced unique challenges related to coin selection and transaction efficiency. This Cardano Problem Statement (CPS) addresses the need for a specialized coin selection approach to optimize transactions involving native tokens while minimizing transaction size and complexity.
+
+## Problem
+
+The integration of native tokens in Cardano transactions has brought both opportunities and challenges. One of the significant challenges is associated with coin selection when dealing with native tokens.
+
+### 1. Need for Efficient Coin Selection
+
+Cardano transactions involving native tokens often require attaching a minUTxO value (lovelace) to the transaction. In scenarios where multiple tokens are associated with a single UTxO, selecting such UTxOs as inputs can lead to inefficient transactions. This inefficiency arises from the increased transaction size due to token information, potentially impacting the decentralized applications and network performance.
+
+### 2. Prioritizing Necessary Tokens
+
+To improve the efficiency of Cardano transactions, there is a need to prioritize the selection of necessary tokens while excluding unnecessary ones. This prioritization will help reduce the inflation of transaction sizes, making them more streamlined and cost-effective.
+
+### 3. Streamlining the Selection Process
+
+In addition to token prioritization, streamlining the selection process is crucial. Enhancements should be made to the selection algorithms to ensure that even in complex scenarios, the coin selection process remains efficient and doesn't compromise the user experience.
+
+### 4. Compatibility with Serialization Libraries
+
+The largest off-chain serialization library, cardano-serialization-lib, still follows a modified version of the CIP-2 standard, which was established in the pre-native token era. There's a need to ensure that the proposed coin selection approach remains compatible with existing serialization libraries, making it accessible to a wider range of developers and applications.
+
+While CIP-2 certainly works well in an environment where native assets such as tokens and NFTs don't exist, it has been expanded upon differently by each serialization library, with their own custom solutions to select for tokens. It would be useful to once again have some standard for coin selection that is trusted by the community.
+
+## Use Cases
+
+### Native Token Transactions
+
+Users and applications frequently engage in native token transactions, making efficient coin selection essential to minimize transaction costs and network congestion.
+
+### DApps and DeFi
+
+Decentralized applications and DeFi projects require efficient coin selection to maintain the performance and cost-effectiveness of their transactions.
+
+### Example Implementation
```

Comment:
Not sure if this is the place to add this block as we discussed in biweekly. Also if anyone knows the right link pointing to the implementation for cardano wallet please help to supplement!

Reply:
I think this makes perfect sense because _Example Implementations_ will always be closely related to, if not qualify as, _Use Cases_.

Reply:
also I would like to note from our meetings the ones that have come up so far:
👉 the dev version of `cardano-wallet` @nicolasayotte mentioned in today's meeting
👉 anything open source from what @siegfried mentioned in https://github.com/cardano-foundation/CIPs/pull/611#issuecomment-1828550747
👉 `cardano-utxo-wasm` as per https://github.com/cardano-foundation/CIPs/pull/611#pullrequestreview-1731138083